src/bloqade/task/base.py

```diff
@@ -95,9 +109,49 @@ def run(self, **kwargs):
         raise NotImplementedError
 
 
-# Report is now just a helper class for
-# organize and analysis data:
 class Report:
+    """Report is a helper class for organizing and analysing data
+
+    ### Analyzing Results
+    When you've retrieved your results from either emulation
+    or hardware you can generate a `.report()`:
+
+    ```python
+    report = results.report()
+    ```
+
+    For the examples below we analyze the results of a two atom program.
+
+    The report contains useful information such as:
+
+    * The raw bitstrings measured per each execution of the program
```

Comment:
I think we can omit bullet points, just plain text is fine

```suggestion
    The raw bitstrings measured per each execution of the program
```